cmd/edge-proxy/main.go

```diff
@@ -131,12 +131,18 @@ func main() {
 	}
 
 	go func() {
-		server.StartHTTPTunnel(httpTunnelAddr, externalHTTPProxyURI)
+		err := server.StartHTTPTunnel(httpTunnelAddr, externalHTTPProxyURI)
+		if err != nil {
```

Comment:
I suggest printing err before exiting.  And below as well.

Reply:
`StartHTTPTunnel()` should print out some error logs when it returns. But yeah, it is always not a bad idea to print out more logs.

Reply:
done